src/main/java/org/prebid/server/geolocation/MaxMindGeoLocationService.java

```diff
@@ -62,6 +62,10 @@ public static MaxMindGeoLocationService create(String dbArchive, String database
 
     @Override
     public Future<GeoInfo> lookup(String ip, Timeout timeout) {
+        if (databaseReader == null) {
+            return Future.failedFuture("Database file does not exist");
```

Comment:
i'd like to change the message to `Geo location database file does not downloaded yet, try again later.`